pkg/cloud/aws/actuators/machine/actuator.go

```diff
@@ -314,11 +316,10 @@ func (a *Actuator) isMachineOutdated(machineSpec *v1alpha1.AWSMachineProviderSpe
 	}
 
 	if aws.BoolValue(machineSpec.PublicIP) != instanceHasPublicIP {
-		return true
+		errs = append(errs, errors.Errorf("(input public IP setting \"%v\" != existing \"%v\")", aws.BoolValue(machineSpec.PublicIP), instanceHasPublicIP))
```

Comment:
```suggestion
		errs = append(errs, errors.Errorf("(public IP setting cannot be mutated from \"%v\" to \"%v\")", instanceHasPublicIP, aws.BoolValue(machineSpec.PublicIP)))
```

Reply:
thanks for these! I like your wording way more than mine.